apps.gen.go

```diff
@@ -9,20 +9,21 @@ import (
 
 // App An application's configuration and status.
 type App struct {
-	ID                      string      `json:"id,omitempty"`
-	OwnerUUID               string      `json:"owner_uuid,omitempty"`
-	Spec                    *AppSpec    `json:"spec"`
-	DefaultIngress          string      `json:"default_ingress,omitempty"`
-	CreatedAt               time.Time   `json:"created_at,omitempty"`
-	UpdatedAt               time.Time   `json:"updated_at,omitempty"`
-	ActiveDeployment        *Deployment `json:"active_deployment,omitempty"`
-	InProgressDeployment    *Deployment `json:"in_progress_deployment,omitempty"`
-	LastDeploymentCreatedAt time.Time   `json:"last_deployment_created_at,omitempty"`
-	LiveURL                 string      `json:"live_url,omitempty"`
-	Region                  *AppRegion  `json:"region,omitempty"`
-	TierSlug                string      `json:"tier_slug,omitempty"`
-	LiveURLBase             string      `json:"live_url_base,omitempty"`
-	LiveDomain              string      `json:"live_domain,omitempty"`
+	ID                      string       `json:"id,omitempty"`
+	OwnerUUID               string       `json:"owner_uuid,omitempty"`
+	Spec                    *AppSpec     `json:"spec"`
+	DefaultIngress          string       `json:"default_ingress,omitempty"`
+	CreatedAt               time.Time    `json:"created_at,omitempty"`
+	UpdatedAt               time.Time    `json:"updated_at,omitempty"`
+	ActiveDeployment        *Deployment  `json:"active_deployment,omitempty"`
+	InProgressDeployment    *Deployment  `json:"in_progress_deployment,omitempty"`
+	LastDeploymentCreatedAt time.Time    `json:"last_deployment_created_at,omitempty"`
+	LiveURL                 string       `json:"live_url,omitempty"`
+	Region                  *AppRegion   `json:"region,omitempty"`
+	TierSlug                string       `json:"tier_slug,omitempty"`
+	LiveURLBase             string       `json:"live_url_base,omitempty"`
+	LiveAppDomain           string       `json:"live_domain,omitempty"`
```

Comment:
This looks like a breaking change – could we keep the old `LiveDomain` field and add a comment to say that it's deprecated, so that users who upgrade godo and depend on this aren't surprised with compilation errors? Something like,

```suggestion
	// LiveDomain is deprecated. Please use LiveAppDomain instead.
	LiveDomain              string       `json:"-"`
	LiveAppDomain           string       `json:"live_domain,omitempty"`
```

could work I think.

Reply:
Ah good catch! I think this is a mistake with the autogen code. Not sure what caused it, but it has to be a recent change. I'll look into it and fix it so it's back to `LiveDomain`

Reply:
@bentranter fixed